code/src/index.css

```diff
@@ -11,3 +11,210 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+  .start-container { 
+  background-image: url("./images/labyrinth_start_map.jpg"); 
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+} 
+
+.start-wrapper {
+  display: grid;
+  justify-content: center;
+  justify-items: center;
+  width: 370px;
+  background-color: rgba(248, 248, 248, 0.8);
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.3);
+  padding: 2vh 2vh 4vh 2vh;
+} 
+
+.start-wrapper h1 {
+  font-size: 38px;
+  font-weight: 900;
+  font-style: italic;
+  margin: 0;
+  padding-bottom: 1vh;
+}
+
+.start-wrapper h2 {
+font-size: 32px;
+line-height: 36px;
+font-weight: 600;
+margin: 0;
+padding-left: 2vh;
+padding-right: 2vh;
+}
+
+.start-wrapper p {
+font-size: 24px;
+line-height: 32px;
+font-weight: 500;
+padding-left: 2vh;
+padding-right: 2vh;
+}
+
+.text-field {
+  width: 250px;
+  height: 39px;
+  background-color: #ffffff;
+  border: 3px solid #DBD4D4;
+  box-sizing: border-box;
+  border-radius: 5px;
+  box-shadow: 0 4px 4px rgba(0,0,0,0.25);
+}
+
+button {
+  min-width: 120px;
+  min-height: 50px;
+  margin-top: 2vh;
+  background-color: #E9B550;
+  border: solid 4px #E8A625;
+  border-radius: 30px;
+  box-shadow: 0 4px 4px 0 #bdbdbc;
+}
+
+button {
+  font-size: 18px;
+  font-family: 'Playfair Display';
+  font-style: italic;
+  font-weight: bold;
+  margin: 2vh 1vh 0 1vh;
+}
```

Comment:
we were wondering  a bit here about the two button classes with same name, does this work?